docker/README.md

```diff
@@ -22,3 +22,25 @@ docker run ucx-py
 ```
 
 The container above will run UCX-Py tests and benchmarks.
+
+## Infiniband/NVLink-enabled docker file
+
+In addition to the reference Docker image, the `UCXPy-CUDA.dockerfile` builds an image with support for CUDA and MOFED. This is based on the [nvidia/cuda:11.5.2-devel-ubuntu20.04](https://hub.docker.com/layers/cuda/nvidia/cuda/11.5.2-devel-ubuntu20.04/images/sha256-fed73168f35a44f5ff53d06d61a1c55da7c26e7ca5a543efd78f35d98f29fd4a?context=explore) image, and uses MOFED version 5.3-1.0.5.0. To function successfully the OFED version reported by `ofed_info -s` on the host should match this version.
+
+To use this image
+```bash
+docker build -t ucx-py-ib -f UCXPy-CUDA.dockerfile .
+```
+
+### Running
+
+To expose high-performance transports from the host in the container requires a number of additional flags when running the container. `docker run --privileged` is a catch-all that will definitely provide enough permissions (`ulimit -l unlimited` is then needed in the container). Alternately, provide `--ulimit memlock=-1` and expose devices with `--device /dev/infiniband/uverbs$X` (where `$X` is replaced by an appropriate device number, once per device), see [the UCX documentation](https://openucx.readthedocs.io/en/master/running.html#running-in-docker-containers) for more details.
+
+For example, a run command that exposes all devices on a DGX-1 is:
+
+```bash
+docker run --ulimit memlock=-1 --device /dev/infiniband/uverbs0 --device /dev/infiniband/uverbs1 \ 
+  --device /dev/infiniband/uverbs2 --device /dev/infiniband/uverbs3 -ti ucx-py-ib /bin/bash
```

Comment:
Perhaps we could map just `--device /dev/infiniband`?

Reply:
Yes, this works.